Lib/fontTools/merge/__init__.py

```diff
@@ -210,17 +99,11 @@ def merge(self, fontfiles):
 		self.fonts = fonts
 		self.duplicateGlyphsPerFont = [{} for _ in fonts]
 
+		compute_mega_cmap(self, [font['cmap'] for font in fonts])
+
 		allTags = reduce(set.union, (list(font.keys()) for font in fonts), set())
 		allTags.remove('GlyphOrder')
 
-		# Make sure we process cmap before GSUB as we have a dependency there.
```

Comment:
you got rid of this block that places cmap before GSUB in allTags list? Is it safe and the code/comment above was stale?

Reply:
Yes. I restructured the code so it's not needed anymore.

Reply:
Because I'm calling `computeMegaCmap` explicitly now.